packages/shared-core/src/sdk/documents/screens.ts

```diff
@@ -0,0 +1,23 @@
+import { Screen } from "@budibase/types"
+
+export function findInSettings(screen: Screen, toFind: string) {
+  const foundIn: { setting: string; value: string }[] = []
+  function recurse(props: Record<string, any>, parentKey = "") {
+    for (let key of Object.keys(props)) {
```

Comment:
```suggestion
    for (const key of Object.keys(props)) {
```

Also, given you're not reassigning `props[key]`, it might be a bit nicer to use `for (const [key, value] of Object.entries(props))`.